docs/learn/guides/integration-patterns/backend-for-frontend/src/message_mgt/message_mgt_service.bal

```diff
@@ -112,21 +110,21 @@ service message_mgt_service on httpListener {
         http:Response response = new;
 
         // Create a json array with Messages
-        json messageResponse = { Messages: [] };
+        map<json> messageResponse = { Messages: [] };
 
         // Get all Messages from map and add them to response
         int i = 0;
-        foreach var(k, v) in messageMap {
-            json messageValue = v.Message;
+        foreach var v in messageMap {
+            json messageValue = <json>v.Message;
             string messageStatus = messageValue.Status.toString();
             if (messageStatus == "Unread"){
-                messageResponse.Messages[i] = messageValue;
```

Comment:
If the variable i's usage is removed, we can remove that variable itself?

Reply:
Method has been removed in the latest version : https://github.com/wso2/ballerina-integrator/blob/master/docs/content/src/learn/tutorials/integration-patterns-and-soa/backend-for-frontend/src/healthcare_management_service/downstream_services/message_mgt_service.bal